### PR TITLE
Warning on tenant usage

### PR DESCRIPTION
**Pull request by philippelaumay**
add explicit message on tenant usage and limitation for customers

### CODE REVIEW

File: modules/ROOT/pages/multi-tenancy-and-tenant-configuration.adoc [hunk 1]
```diff
@@ -3,14 +3,23 @@
 
 [WARNING]
 ====
+This feature was deprecated in Bonita v2021.1. This means that Multi-tenant architecture will continue to work in Bonita for existing Bonita subscription customers,
+but it will not be offered to new customers, nor offered as a modification for existing Bonita subscription customers who are not already using multi-tenancy.
+====
+
+[NOTE]
+====
+Important note: As mentioned in the release note, Bonita Enterprise customers need a contractual agreement to create additional tenants. For any new tenant creation, any general or contractual question related to Multi-tenancy, please contact your customer success representative.
```
Comment (tbouffard) [inline modules/ROOT/pages/multi-tenancy-and-tenant-configuration.adoc]: about 'important note', remove or put bold. Currently this is hidden

image

File: modules/ROOT/pages/multi-tenancy-and-tenant-configuration.adoc [hunk 2]
```diff
@@ -3,14 +3,23 @@
 
 [WARNING]
 ====
+This feature was deprecated in Bonita v2021.1. This means that Multi-tenant architecture will continue to work in Bonita for existing Bonita subscription customers,
```
Comment (tbouffard) [inline modules/ROOT/pages/multi-tenancy-and-tenant-configuration.adoc]:
```suggestion
This feature was deprecated in Bonita 2021.1. This means that Multi-tenant architecture will continue to work in Bonita for existing Bonita subscription customers,
```

Comment (tbouffard) [reply modules/ROOT/pages/multi-tenancy-and-tenant-configuration.adoc]: we never prefix version with `v` in other parts of the documentation

File: modules/api/pages/platform-api.adoc [hunk 3]
```diff
@@ -125,6 +125,17 @@ Stop current node :
 
 == Tenant
 
+[WARNING]
+====
+This feature was deprecated in Bonita v2021.1.
```
Comment (tbouffard) [inline modules/api/pages/platform-api.adoc]:
```suggestion
This feature was deprecated in Bonita 2021.1. 
```

File: modules/api/pages/platform-api.adoc [hunk 4]
```diff
@@ -125,6 +125,17 @@ Stop current node :
...
 == Tenant
 
+[WARNING]
+====
+This feature was deprecated in Bonita v2021.1. 
+====
+
+[NOTE]
+====
+Important note:  Bonita Enterprise customers need a contractual agreement to create additional tenants. For any new tenant creation, any general or contractual question related to Multi-tenancy, please contact your customer success representative.
```
Comment (tbouffard) [inline modules/api/pages/platform-api.adoc:135]: `Important note` in bold or removed (not visible for now)

File: modules/identity/pages/tenant-admin-credentials.adoc [hunk 5]
```diff
@@ -2,8 +2,12 @@
 :page-aliases: ROOT:tenant_admin_credentials.adoc, ROOT:tenant-admin-credentials.adoc
 :description: Each tenant has an administrator (also known as the *tenant technical user*) with a tenant-specific username and password.
 
-{description}
+[WARNING]
+====
+Important note: Bonita Enterprise customers need a contractual agreement to create additional tenants. For any new tenant creation, any general or contractual question related to Multi-tenancy, please contact your customer success representative.
```
Comment (tbouffard) [inline modules/identity/pages/tenant-admin-credentials.adoc]: `Important note` in bold or removed (not visible for now)

File: modules/identity/pages/tenant-admin-credentials.adoc [hunk 6]
```diff
@@ -2,8 +2,12 @@
...
+====
 
+{description}
```
Comment (tbouffard) [inline modules/identity/pages/tenant-admin-credentials.adoc:10]:
```suggestion
{description}

```

File: modules/ROOT/pages/multi-tenancy-and-tenant-configuration.adoc [hunk 7]
```diff
@@ -3,14 +3,23 @@
...
+but it will not be offered to new customers, nor offered as a modification for existing Bonita subscription customers who are not already using multi-tenancy.
+====
+
+[NOTE]
```
Comment (alachambre) [inline modules/ROOT/pages/multi-tenancy-and-tenant-configuration.adoc]:
```suggestion
[IMPORTANT]
```

File: modules/ROOT/pages/multi-tenancy-and-tenant-configuration.adoc (hunk 1 above)
Comment (alachambre) [inline modules/ROOT/pages/multi-tenancy-and-tenant-configuration.adoc]:
```suggestion
As mentioned in the release note, Bonita Enterprise customers need a contractual agreement to create additional tenants. For any new tenant creation, any general or contractual question related to Multi-tenancy, please contact your customer success representative.
```

Comment (alachambre) [reply modules/ROOT/pages/multi-tenancy-and-tenant-configuration.adoc]: The IMPORTANT admonition is here for that...

File: modules/identity/pages/tenant-admin-credentials.adoc [hunk 8]
```diff
@@ -2,8 +2,12 @@
 :page-aliases: ROOT:tenant_admin_credentials.adoc, ROOT:tenant-admin-credentials.adoc
 :description: Each tenant has an administrator (also known as the *tenant technical user*) with a tenant-specific username and password.
 
-{description}
+[WARNING]
```
Comment (alachambre) [inline modules/identity/pages/tenant-admin-credentials.adoc:5]:
```suggestion
[IMPORTANT]
```

File: modules/identity/pages/tenant-admin-credentials.adoc (hunk 5 above)
Comment (alachambre) [inline modules/identity/pages/tenant-admin-credentials.adoc]:
```suggestion
Bonita Enterprise customers need a contractual agreement to create additional tenants. For any new tenant creation, any general or contractual question related to Multi-tenancy, please contact your customer success representative.
```

**Review by alachambre — CHANGES_REQUESTED** [review]
Update your admonitions to 'important' ones if the content is important please. If you use an other amdonition and write 'important' at the beginning it doesn't make sens and lead to confusion.

File: modules/ROOT/pages/multi-tenancy-and-tenant-configuration.adoc (hunk 7 above)
Comment (tbouffard) [reply modules/ROOT/pages/multi-tenancy-and-tenant-configuration.adoc]: I agree with the proposal as it clarifies the intention. In this case, we can remove 'Important Note' from the text
See https://docs.asciidoctor.org/asciidoc/latest/blocks/admonitions/

**Pull request merged**